ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,418 @@
+
+``` text
+- DC1
+  - DC1_FABRIC
+    - DC1_SPINES
+    - DC1_LEAFS
+  - DC1_NETWORK_SERVICES
+    - DC1_SPINES
+    - DC1_LEAFS
+  - DC1_NETWORK_PORTS
+    - DC1_SPINES
+    - DC1_LEAFS
+```
+
+DC1 represents the highest level within the hierarchy. Ansible variables defined at this level will be applied to all nodes in the fabric. Ansible groups have parent and child relationships. For example, both DC1_SPINES and DC1_LEAFS are children of DC1_FABRIC. Groups of Groups are possible and allow variables to be shared at any level within the hierarchy. For example, DC1_NETWORK_SERVICES is a group with two other groups defined as children: DC1_SPINES and DC1_LEAFS. The same applies to the group named DC1_NETWORK_PORTS. You will see these groups listed at the bottom of the inventory file.
+
+This naming convention makes it possible to extend anything quickly but can be changed based on your preferences. The names of all groups and hosts must be unique.
+
+![Figure: 2](images/ansible_groups.svg)
+
+### inventory.yml
+
+The below inventory file represents two Spines and four Leafs. The nodes are defined under the groups DC1_SPINES and DC1_LEAFS, respectively. We apply group variables (group_vars) to these groups to define their functionality and configurations.
+
+The hostnames specified in the inventory must exist either in DNS or in the hosts file on your Ansible host to allow successful name lookup and be able to reach the switches directly. A successful ping from the Ansible host to each inventory host verifies name resolution (e.g., ping SPINE1).
+
+Alternatively, if DNS is not available, define the variable ansible_host to be an IP address for each device.
+
+```yaml
+--8<--
+examples/l2ls-fabric/inventory.yml
+--8<--
+```
+
+## AVD Fabric Variables
+
+To apply AVD variables to the nodes in the fabric, we make use of Ansible group_vars. How and where you define the variables is your choice. The group_vars table below is one example of AVD fabric variables.
+
+| group_vars/              | Description                                   |
+| ------------------------ | --------------------------------------------- |
+| DC1.yml                  | Global settings for all devices               |
+| DC1_FABRIC.yml           | Fabric, Topology, and Device settings         |
+| DC1_SPINES.yml           | Device type for Spines                        |
+| DC1_LEAFS.yml            | Device type for Leafs                         |
+| DC1_NETWORK_SERVICES.yml | VLANs                                         |
+| DC1_NETWORK_PORTS.yml    | Port Profiles and Connected Endpoint settings |
+
+The tabs below show the Ansible **group_vars** used in this example.
+
+=== "DC1"
+    At the top level (DC1), the following variables are defined in **group_vars/DC1.yml**. These Ansible variables apply to all nodes in the fabric and is a common place to set AAA, users, NTP, and management interface. Update local_users and passwords for your environment.
```

Comment:
Consider ending sentence with settings/variables. Example "These Ansible variables apply to all nodes in the fabric and is a common place to set AAA, users, NTP, and management interface settings."

Reply:
fixed